### ISSUE TITLE
hiveadmission logs showing authentication error: x509: certificate signed by unknown authority

### ISSUE THREAD

**Issue by chrisahl** [openshift/hive#827]
OCP 4.2 (1 master, 3 worker) running in IBM Fyre environment
OCP was deployed and less than one hour later MCM (with hive pulled from master in a personal build and PR I am working on) was installed.  Less than 24 hours later,  Dominique was attempting to do some hive operations and noticed the following:

`oc logs hiveadmission-949dfdff6-mm6k8 -n hive`

```
Error from server (InternalError): error when creating "STDIN": Internal error occurred: failed calling webhook "clusterimagesetvalidators.admission.hive.openshift.io": the server is currently unable to handle the request
[HIVE INSTALL] Create image set failed
 [HIVE INSTALL] FAILURE: Failed to create image set
```
and tons of 
```
Error in admin webhook:
E0213 12:26:19.887956       1 authentication.go:65] Unable to authenticate the request due to an error: x509: certificate signed by unknown authority
```

@dgoodwin noted the following in slack channel mgmt-collab-guest (https://ibm-cloudplatform.slack.com/archives/GNXFTBZCL/p1581597401090400)

```
dgoodwin-4049  2 hours ago
whats the environment like where hive was running?

dgoodwin-4049  2 hours ago
and how long had that cluster been in existence when hive was installed?

dgoodwin-4049  2 hours ago
we might need you to file a github issue for that, i will likely need the teams help to debug and i dont think all of them are on this slack

dgoodwin-4049  2 hours ago
openshift rotates certs after something like 24 hours, i believe we've seen it cause issues like this if hive was installed in the first 24 on a openshift 4 cluster

dgoodwin-4049  2 hours ago
its not something we see often because all our prod hive envs we still run on 3.11, and developers would rarely leave a cluster up more than a day

dgoodwin-4049  2 hours ago
so may be an important discovery
```

**Comment by chrisahl** [comment]
FYI... @itdove

**Comment by dgoodwin** [comment]
So Hive installed within 24 hours, but the error surfaced within 24 hours also.

What is this Fyre environment?

Does deleting the hiveadmission pods fix the problem?

**Comment by itdove** [comment]
Fyre is an infra we use to deploy OCP.

Does deleting the hiveadmission pods fix the problem? yes

**Comment by dgoodwin** [comment]
Ok that's very good, strong indication of what we thought might be going on. Still weird it's not happening after a 24h though.

**Comment by staebler** [comment]
Fixed by #830.

/close
